src/common/Rates/index.js

```diff
@@ -22,7 +22,8 @@ export const Rates = ({ mainInfo, hideTotalScore, voteAverage, voteCount }) => {
                                 {Number(voteAverage).toFixed(1).replace('.', ',')}
                                 <TotalScore $hideTotalScore={isTotalScoreHidden} $larger={isRatesAreMainInfo}>/ 10</TotalScore>
                             </Score>
-                            <Votes $larger={isRatesAreMainInfo}>{Number(voteCount)} votes</Votes>
+                            <Votes $larger={isRatesAreMainInfo}>
+                                {Number(voteCount).toLocaleString().replace(/,/g, ' ')} votes</Votes>
```

Comment:
Hey, that is good idea to sepearte numbers, with spaces but in this case design requires a comma.
Here my suggestion:

- **Number(voteCount).toLocaleString().replace(/,/g, ' ,')**
This code replace comma in vote with space on the left and comma.
In my opinion it more matches the design

Reply:
In the design it doesn't show exactly what the number of votes above 1000 should look like - it seems to me that the version with a space is more universal for Europe - but globally the version with a comma will actually be better :ok_hand: